interface-definitions/vpn_ipsec.xml.in

```diff
@@ -978,6 +979,45 @@
                       </constraint>
                     </properties>
                   </leafNode>
+                  <node name="range">
```

Comment:
I'm not a big fan of ranges, is there a valid reason to not use a prefix?

Reply:
The main reason to offer range is if the user wants to give the router its own address on the VTI interface subnet.

E.g. this is how I have mine configured:
```
 vti vti2 {
     address 10.23.58.1/24
     address fdcc:2200:a8ee:2358::6970:1/112
     description "Client VPN"
     mtu 1416
 }
 ```

The range is needed to keep strongswan from giving out the `fdcc:2200:a8ee:2358::6970:1` address assigned to VyOS. I use the router's address in this subnet to hand out as a DNS resolver to connected VPN clients.

There are certainly other ways to set the network up, but keeping services in the same subnet as the clients can be convenient especially for any sort of split tunneling setup where you might want to minimize the number of routes you need to push to the client.